tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -17,9 +17,32 @@ use testapi;
 use base "reboot_and_wait_up";
 
 sub run {
-    my $self    = shift;
-    my $timeout = 600;
+    my $self = shift;
+
+    #initialized to be offline upgrade
+    my $timeout = 180;
     set_var("reboot_for_upgrade_step", "yes");
+    set_var("offline_upgrade",         "yes");
```

Comment:
These runtime vars should be uppercase?

Reply:
Really? runtime vars are dynamic, aren't constant settings more likely to be upper case? 
If it is not well followed by everyone, then I hope to keep it as it is now, can I?

Reply:
That's fine :)